packages/x-adapter-platform/src/__tests__/__fixtures__/identifier-results.response.ts

```diff
@@ -3,8 +3,8 @@ export const platformIdentifierResultsResponse = {
   catalog: {
```

Comment:
Be careful with this, it seems that the top-clicked response is not standardized. We are going to wait until the response will be standarized.